src/app.ts

```diff
@@ -9,11 +9,12 @@
 import fs = require('fs');
 import Discord = require('discord.js');
 import { BaseCommand } from './command_base';
+import { AppServices } from './appServices';
 
 export type ID = '' | string & {__isID: true};
 
 interface Constructable<T> {
-	new(message: Discord.Message): T;
+	new(message: Discord.Message, services: AppServices): T;
```

Comment:
```suggestion
	new(message: Discord.Message, services?: AppServices): T;
```
See other suggestion for details on this change.